projects/reservoir-protocol/index.js

```diff
@@ -1,22 +1,31 @@
 const config = {
-  ethereum: [
-    // https://docs.reservoir.xyz/products/proof-of-reserves
-    '0x0c7e4342534e6e8783311dCF17828a2aa0951CC7',
-    '0x9BB2c38F57883E5285b7c296c66B9eEA4769eF80',
-    '0x99A95a9E38e927486fC878f41Ff8b118Eb632b10',
-    // '0x31Eae643b679A84b37E3d0B4Bd4f5dA90fB04a61', - exluded RUSD because it is project's own token
-  ]
+    ethereum: [
+        // https://docs.reservoir.xyz/products/proof-of-reserves
+        '0x0c7e4342534e6e8783311dCF17828a2aa0951CC7',
+        '0x9BB2c38F57883E5285b7c296c66B9eEA4769eF80',
+        '0x99A95a9E38e927486fC878f41Ff8b118Eb632b10',
+        '0x31Eae643b679A84b37E3d0B4Bd4f5dA90fB04a61'
+    ]
 }
 
 Object.keys(config).forEach(chain => {
-  const funds = config[chain]
-  module.exports[chain] = {
-    tvl: async (api) => {
-      const tokens = await api.multiCall({ abi: 'address:underlying', calls: funds })
-      const bals = await api.multiCall({ abi: 'uint256:totalValue', calls: funds })
-      const decimals  = await api.multiCall({  abi: 'uint8:decimals', calls: tokens })
-      bals.forEach((v, i) => bals[i] = v * 10 ** (decimals[i] - 18))
-      api.add(tokens, bals)
+    const funds = config[chain]
+    module.exports[chain] = {
+        tvl: async (api) => {
+            const tokens = await api.multiCall({ abi: 'address:underlying', calls: funds })
+            const bals = await api.multiCall({ abi: 'uint256:totalValue', calls: funds })
+            const decimals = await api.multiCall({ abi: 'uint8:decimals', calls: tokens })
+            bals.forEach((v, i) => bals[i] = v * 10 ** (decimals[i] - 18))
+            api.add(tokens, bals)
+
+            const psmBal = await api.call({ abi: 'erc20:balanceOf', target: '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48', params: ['0x4809010926aec940b550D34a46A52739f996D75D'] })
```

Comment:
added it here: https://github.com/DefiLlama/DefiLlama-Adapters/commit/8b5000b9d808cd5a9fa8536d0d3a26b93530b8d4

Reply:
Will this PR get merged? Or are the changes simply in the commit you shared?

Reply:
@g1nt0ki are there any outstanding issues or can this be merged?